mission/para_player_init_client.sqf

```diff
@@ -92,9 +92,6 @@ progressLoadingScreen 1.0;
 //Setup teleporters
 call vn_mf_fnc_action_teleport;
 
-call vn_mf_fnc_apply_unit_traits;
-
-call vn_mf_fnc_action_trait;
```

Comment:
these are handled by the call to `vn_mf_fnc_change_team` within `para_player_init_server.sqf`

after a double check, this might cause an issue for missions where the player spawn location is not in range of a duty officer?

```sqf
// ./mission/functions/core/teams/fn_change_team.sqf
// ...
if (vn_mf_duty_officers inAreaArray [getPos _player, 20, 20, 0, false, 20] isEqualTo []) exitWith {
        ["TaskFailed",["","STR_vn_mf_needdutyofficer"]] remoteExecCall ["para_c_fnc_show_notification",_player];
        false
};
```